src/app/components/tools/Button.jsx

```diff
@@ -0,0 +1,9 @@
+const Button = ({ text, className }) => (
```

Comment:
please rename the folder name from tools to something more appropriate like common/form/etc